### PR TITLE
Fix user group permission and creates a modal for user group removal

### PR DESCRIPTION
**Pull request by Leandroqo**
## Issue Description

It shows status `403` when a maintainer removes a user group that it belongs to.

## Solution

- creates a modal that asks the user about the removal of a user group.

Captura de Tela (363)

### CODE REVIEW

File: ui/src/modules/Settings/Credentials/__tests__/Credentials.spec.tsx [hunk 1]
```diff
@@ -60,6 +76,7 @@ test('render Credentials default component', async () => {
   );
 
   render(<Credentials />);
+  screen.debug()
```
Comment (bCatanant) [inline ui/src/modules/Settings/Credentials/__tests__/Credentials.spec.tsx:79]: Remove screen.debug()

**Review by bCatanant — CHANGES_REQUESTED** [review]

**Review by bCatanant — APPROVED** [review]

**Review by bCatanant — APPROVED** [review]

File: ui/src/core/components/Modal/Trigger/styled.ts [hunk 2]
```diff
@@ -60,12 +60,16 @@ const Content = styled.div`
 
 const Title = styled(Text.h2)`
   text-align: left;
-  margin-bottom: 20px;
+  margin-bottom: 0;
+`;
+
+const Description = styled.div`
+  margin: 20px 0 20px 0;
```
Comment (luanecavalcantizup) [inline ui/src/core/components/Modal/Trigger/styled.ts:67]: As confirmed with the UX (Jose Silva), the margins top and bottom of the text description of trigger modal is 20px.

File: ui/src/core/components/Modal/Trigger/styled.ts [hunk 3]
```diff
@@ -79,14 +83,15 @@ const ContinueButton = styled(DefaultButton)`
 const DismissButton = styled(DefaultButton)`
   cursor: pointer;
   padding: 5px;
-  margin-top: 15px;
+  margin-top: 0;
```
Comment (luanecavalcantizup) [inline ui/src/core/components/Modal/Trigger/styled.ts:86]: I removed the margin-top of the buttons to adjust the margins inside this modal.

**Review by bCatanant — DISMISSED** [review]

**Review by ifdouglas — DISMISSED** [review]

**Comment by luanecavalcantizup** [comment]
blocked by pr #1108

**Comment by luanecavalcantizup** [comment]
waiting to be tested

**Review by bCatanant — APPROVED** [review]

**Pull request merged**